cmake/onnxruntime_unittests.cmake

```diff
@@ -1098,10 +1098,12 @@ if (onnxruntime_BUILD_JAVA)
     set_property(TEST onnxruntime4j_test APPEND PROPERTY DEPENDS onnxruntime4j_jni)
 endif()
 
+# limit to only test on windows first, due to a runtime path issue on linux
```

Comment:
You may copy [this line](https://github.com/microsoft/onnxruntime/blob/master/cmake/onnxruntime_python.cmake#L152) to cmake/onnxruntime_unittests.cmake for the new shared lib you added. Then this test will work on Linux. You can do it in another PR if you want.

Reply:
thanks, i will merge in another pr. let's fix the break first.